glossary.md

```diff
@@ -45,11 +45,14 @@ The maintainer of a shared library which is a dependency of many applications or
 
 ### Feature Flag SDK
 
-The interfaces and abstractions used by Application Author to implement feature flags in their application or service.
+The libraries used by Application Author to implement feature flags in their application or service. The interface of these libraries adhere to the Feature Flag API.
```

Comment:
I like this point. I think it hits on the fact that that SDK is really a library that contains interfaces and abstractions, whereas the